<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>797</width>
    <height>703</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Suche</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="twAnzeige_suche">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>771</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbSuche">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>16</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Suche Fachrichtung:</string>
   </property>
   <widget class="QPushButton" name="btSuchen">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>45</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Suchen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_BEENDEN">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>45</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblFachrichtungbez2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>53</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezeichnung:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFahrichtungabk2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>53</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fachrichtung:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFachrichtungabk2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFachrichtungbez2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
